ftd/t/js/80-or-type-constant.ftd

```diff
@@ -0,0 +1,31 @@
+-- or-type button-type:
+
+-- constant integer small: 1
+
+-- constant integer medium: 2
+
+-- constant integer large: 3
+
+-- end: button-type
+
+
+-- button-type b: small
+
+;; A variable with the same name as a variant of a button type
+;; if we comment this out, the value of "small" will be inferred from the right-hand side or-type
+-- button-type small: small
+
+;; Since the specificity of a variable is higher, the value of the variable will be used
+;; and smol will be printed
+-- ftd.text: !Smol
+text if { b == small }: Smol
```

Comment:
```suggestion
-- ftd.text: !Smol
text if { b == small }: Smol

-- ftd.text: not medium
if: { b != medium }
```

Reply:
@harshdoesdev make test pass after accepting this suggestion.

Reply:
Ok, got it

Reply:
@amitu I have updated the tests and the tests are passing